ullyses/calibrate_stis_data.py

```diff
@@ -385,27 +390,30 @@ def make_plots(self):
         Plot calibration output.
         """
 
+        print("\n", f" CREATING DIAGNOSTIC PLOTS ".center(NCOLS, SYM), "\n")
+        self.plots_made = True
         if self.x1d_mast is not None:
             self.plots_made = True
```

Comment:
Don't need this line any more

Reply:
Good catch! I made the first declaration of `self.plots_made` to be False as a workaround. Even though it's always set to True for now, there may be a future case when I need to leave it as False, so I left the check in for now.